system/types/app_settings.go

```diff
@@ -50,6 +50,15 @@ type (
 				// Can users reset their passwords
 				PasswordReset struct{ Enabled bool } `kv:"password-reset"`
 
+				// PasswordCreate setting for create password for user via generated link with token
+				// If user has no password then link redirects to create password page
+				// Otherwise it redirects to profile page of that user
+				// link can be generated through useradd cli command with `make-password-link` flag
+				PasswordCreate struct {
```

Comment:
This needs to be supported in the admin panel as well.
Can you add a task for next sprint with instructions so one of the frontend devs can pick it up (as an alternative to you doing it ;))